src/main/java/core/basesyntax/dao/FruitTransactionDao.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+public interface FruitTransactionDao {
+    void add(String fruit, int quantity);
+    void update(String fruit, int quantity);
+    int get (String fruit);
```

Comment:
The naming of the interface and methods should clearly reflect their responsibilities. For example, if this interface is meant for handling fruit transactions, consider renaming it to something more specific like 'FruitStockDao'. Additionally, the method names 'add', 'update', and 'get' are too generic. They should describe the action they perform on the fruit transactions, such as 'addTransaction', 'updateTransactionQuantity', and 'getQuantityByFruit' respectively.